datalad_tabby/resources/contexts/common.jsonld

```diff
@@ -10,5 +10,23 @@
   "last-updated": "schema:dateModified",
   "license": "schema:license",
   "name": "schema:name",
+  "url": "schema:url",
   "version": "schema:version",
+  "address": "",
+  "affiliation": "",
+  "annex-key": "",
+  "author": "",
+  "citation": "",
+  "checksum[md5]": "",
+  "contact": "",
+  "date": "",
```

Comment:
Does this refer to a general `date` property or to the datatype (e.g. https://schema.org/Date)

Reply:
The question alone is an indication that this name is not good. It is presently used for publications. It better becomes

https://schema.org/datePublished

Reply:
we could also consider a naming schema with the square brackets similar to `project[code]` or `sample[organism]`, where the part in brackets defines a specific type/flavor/category of date, e.g.: 
```
"date[published]": "schema:datePublished",
"date[start]": "schema:startDate"
```
or these could just be semantically separate keys.